helm/cluster-vsphere/templates/helmreleases/cilium-helmrelease.yaml

```diff
@@ -66,17 +66,6 @@ spec:
             effect: "NoSchedule"
             operator: "Equal"
             value: "true"
-{{- if not .Values.internal.ciliumNetworkPolicy.enabled }}
-    defaultPolicies:
-      enabled: true
-      tolerations:
-        - effect: NoSchedule
-          operator: Exists
-        - effect: NoExecute
-          operator: Exists
-        - key: CriticalAddonsOnly
-          operator: Exists
-{{- end }}
     extraPolicies:
```

Comment:
probably would make sense to remove this block too

Reply:
Yeah :)